### PR TITLE
MTDSA-15491: Update Changelog for Individual-calculations-api

### PR DESCRIPTION
**Pull request by LouiseWong98**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,6 +56,19 @@ Updated Endpoint: `Create CIS Deductions for Subcontractor`
...
+* Version 2.0 has been deprecated see [sandbox changelog](https://github.com/hmrc/income-tax-mtd-changelog/wiki#9th-january-2023)
+* New Gov-Test-Scenario header values added for `Trigger a Self Assessment Tax Calculation` endpoint [sandbox changelog](https://github.com/hmrc/income-tax-mtd-changelog/wiki#1-december-2022)
+* New fields have been added for `Retrieve a Self Assessment Tax Calculation` endpoint [sandbox changelog](https://github.com/hmrc/income-tax-mtd-changelog/wiki#5th-january-2023)
+* New error code has been added for `Submit a Self Assessment Final Declaration` endpoint [sandbox changelog](https://github.com/hmrc/income-tax-mtd-changelog#8-march-2023)
+
+Updated Endpoint: `List Self Assessment Tax Calculations`
+
+* Updated description for the endpoint
+* Updated description for `calculationTimestamp` field
+* Removed `biss` from `	calculationType` field
```
Comment (mattstephens-hmrc) [inline README.md:69]: I'm not sure what that means. Was it previously "bissCalculationType"?  If so, maybe: "Renamed the bissCalculationType field to calculationType"

Comment (mattstephens-hmrc) [reply README.md:69]: (actually there's also currently a space between the tick-mark and calculationType)

File: README.md [hunk 2]
```diff
@@ -56,6 +56,19 @@ Updated Endpoint: `Create CIS Deductions for Subcontractor`
 
 ### individual-calculations-api
 
+* Some error response status codes corrected from 403 to 400, see [sandbox changelog](https://github.com/hmrc/income-tax-mtd-changelog/wiki#10th-january-2023).
+* Version 2.0 has been deprecated see [sandbox changelog](https://github.com/hmrc/income-tax-mtd-changelog/wiki#9th-january-2023)
```
Comment (mattstephens-hmrc) [inline README.md:60]: add a comma after "deprecated" (to be consistent with the previous line)

File: README.md [hunk 3]
```diff
@@ -56,6 +56,19 @@ Updated Endpoint: `Create CIS Deductions for Subcontractor`
...
+* New Gov-Test-Scenario header values added for `Trigger a Self Assessment Tax Calculation` endpoint [sandbox changelog](https://github.com/hmrc/income-tax-mtd-changelog/wiki#1-december-2022)
+* New fields have been added for `Retrieve a Self Assessment Tax Calculation` endpoint [sandbox changelog](https://github.com/hmrc/income-tax-mtd-changelog/wiki#5th-january-2023)
+* New error code has been added for `Submit a Self Assessment Final Declaration` endpoint [sandbox changelog](https://github.com/hmrc/income-tax-mtd-changelog#8-march-2023)
+
+Updated Endpoint: `List Self Assessment Tax Calculations`
```
Comment (mattstephens-hmrc) [inline README.md:65]: Not sure if the endpoint name should have the tick marks, as it isn't "code" as such...  How are we doing the endpoint naming elsewhere in the changelog?

File: README.md (hunk 1 above)
Comment (LouiseWong98) [reply README.md:69]: It's from [here](https://github.com/hmrc/individual-calculations-api/pull/409/files), it used to have `biss` as one of the calculation types but now it has been removed. 
Screenshot 2023-03-28 at 13 02 33

**Review by mattstephens-hmrc — APPROVED** [review]

**Pull request merged**